typedb-src/modules/ROOT/pages/quickstart-guide.adoc

```diff
@@ -0,0 +1,534 @@
+= Quickstart guide
+:keywords: getting started, typedb, typeql, tutorial, quickstart, console, studio
+:longTailKeywords: get started with typedb, typedb tutorial, typedb quickstart, learn typedb
+:pageTitle: Quickstart guide
+:summary: Learn how to create a TypeDB database, load schema and data, perform queries.
+:tabs-sync-option:
+:experimental:
+
+This Quickstart guide shows how to set up a TypeDB database and run queries.
+
+== Prerequisites
+
+[#_download_sample_data]
+=== Download schema and data
+
+For this Quickstart guide, you need to download and use the following files:
+
+[#_iam_schema]
+* link:https://github.com/vaticle/typedb-docs/blob/master/typedb-src/modules/ROOT/attachments/iam-schema.tql[iam-schema.tql,window=_blank]
+  -- The TypeQL define query for the IAM schema.
+//* xref:attachment$iam-schema.tql[iam-schema.tql] -- The TypeQL define query for the IAM schema.
+[#_iam_microdataset]
+* link:https://github.com/vaticle/typedb-docs/blob/master/typedb-src/modules/ROOT/attachments/iam-data.tql[iam-data.tql,window=_blank]
+  -- TypeQL insert queries of a sample dataset for the IAM schema.
+//* xref:attachment$iam-data.tql[iam-data.tql] -- TypeQL insert queries of minimalistic dataset for the IAM schema.
+
+=== Install and run TypeDB
+
+To proceed, please ensure you have TypeDB server (or cluster)
+xref:installation.adoc#_install[installed,window=_blank] and
+xref:installation.adoc#_start_the_server[running,window=_blank].
+
+[#_typedb_client]
+== Select a TypeDB Client to proceed
+
+Choose either TypeDB Console (CLI) or TypeDB Studio (GUI) in the tabs below and complete all the steps.
+
+We recommend familiarizing yourself with at least one of these two Clients, before trying to start developing
+your own application with language-specific TypeDB Drivers.
+
+[tabs]
+====
+TypeDB Console::
++
+--
+TypeDB Console is a CLI TypeDB Client.
+
+image::typedb::quickstart-console.png[TypeDB Console, width = 75%, link=self]
+
+TypeDB Console is usually installed together with a TypeDB server in the same directory.
```

Comment:
I usually run `typedb server` or `typedb console`. Is it useful talk about TypeDB Console as if the user installed it separately?